src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -408,4 +408,44 @@ else if (sibling.getType() == TokenTypes.SINGLE_LINE_COMMENT) {
                 || BlockCommentPosition.isOnPackage(blockComment));
     }
 
+    /**
+     * Gets the content of inline custom tag.
+     *
+     * @param javadocInlineTag javadoc inline tag node.
+     * @return String consisting of the content of inline custom tag.
+     */
+    public static String getContentOfInlineCustomTag(DetailNode javadocInlineTag) {
```

Comment:
can't we just call this `inlineTag` instead of `javadocInlineTag`?

Reply:
Done

Reply:
Resolved.